src/Mvc/Mvc.Testing/src/PublicAPI.Unshipped.txt

```diff
@@ -49,5 +49,6 @@ virtual Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory<TEntryPoint>.Crea
 virtual Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory<TEntryPoint>.CreateHostBuilder() -> Microsoft.Extensions.Hosting.IHostBuilder?
 virtual Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory<TEntryPoint>.CreateServer(Microsoft.AspNetCore.Hosting.IWebHostBuilder! builder) -> Microsoft.AspNetCore.TestHost.TestServer!
 virtual Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory<TEntryPoint>.CreateWebHostBuilder() -> Microsoft.AspNetCore.Hosting.IWebHostBuilder!
+virtual Microsoft.AspNetCore.Mvc.Testing.WebApplicationFactory<TEntryPoint>.DisposeAsyncCore() -> System.Threading.Tasks.ValueTask
```

Comment:
We don't have this method anywhere (and we shouldn't have it)

Reply:
I think the baseline needs to be updated

Reply:
@javiercn tried removing it, but the build fails [like so](https://github.com/dotnet/aspnetcore/pull/32645/checks?check_run_id=2684163922).

Reply:
@javiercn the last changes pass all the builds.